Examples/Swift/ViewController.swift

```diff
@@ -242,6 +242,34 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         present(navigationViewController, animated: true, completion: nil)
     }
     
+    func navigationMapView(_ mapView: MGLMapView, viewFor annotation: MGLAnnotation) -> MGLAnnotationView? {
```

Comment:
This method should be implemented in the extension below that conforms to NavigationViewControllerDelegate.